pkg/security/registry/podsecuritypolicyreview/rest.go

```diff
@@ -0,0 +1,130 @@
+package podsecuritypolicyreview
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/golang/glog"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	"k8s.io/kubernetes/pkg/runtime"
+	kscc "k8s.io/kubernetes/pkg/securitycontextconstraints"
+	"k8s.io/kubernetes/pkg/serviceaccount"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	securityapi "github.com/openshift/origin/pkg/security/api"
+	securityvalidation "github.com/openshift/origin/pkg/security/api/validation"
+	"github.com/openshift/origin/pkg/security/registry/podsecuritypolicysubjectreview"
+	oscc "github.com/openshift/origin/pkg/security/scc"
+)
+
+// REST implements the RESTStorage interface in terms of an Registry.
+type REST struct {
+	sccMatcher oscc.SCCMatcher
+	client     clientset.Interface
+}
+
+// NewREST creates a new REST for policies..
+func NewREST(m oscc.SCCMatcher, c clientset.Interface) *REST {
+	return &REST{sccMatcher: m, client: c}
+}
+
+// New creates a new PodSecurityPolicyReview object
+func (r *REST) New() runtime.Object {
+	return &securityapi.PodSecurityPolicyReview{}
+}
+
+// Create registers a given new PodSecurityPolicyReview instance to r.registry.
+func (r *REST) Create(ctx kapi.Context, obj runtime.Object) (runtime.Object, error) {
+	pspr, ok := obj.(*securityapi.PodSecurityPolicyReview)
+	if !ok {
+		return nil, kapierrors.NewBadRequest(fmt.Sprintf("not a PodSecurityPolicyReview: %#v", obj))
+	}
+	if errs := securityvalidation.ValidatePodSecurityPolicyReview(pspr); len(errs) > 0 {
+		return nil, kapierrors.NewInvalid(securityapi.Kind(pspr.Kind), "", errs)
+	}
+	ns, ok := kapi.NamespaceFrom(ctx)
+	if !ok {
+		return pspr, kapierrors.NewBadRequest("namespace parameter required.")
+	}
+	serviceAccounts, err := getServiceAccounts(pspr.Spec, r.client, ns)
+	if err != nil {
+		return pspr, err
+	}
+
+	if len(serviceAccounts) == 0 {
+		glog.Errorf("No service accounts for namespace %s", ns)
+		return pspr, nil
+	}
+
+	errs := []error{}
+	newStatus := securityapi.PodSecurityPolicyReviewStatus{}
+	for _, sa := range serviceAccounts {
+		userInfo := serviceaccount.UserInfo(ns, sa.Name, "")
+		saConstraints, err := r.sccMatcher.FindApplicableSCCs(userInfo)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("error finding SCC for ServiceAccount %s: %v", sa.Name, err))
```

Comment:
Your using full SecurityContextConstraints in other places, maybe you should do here, as well.